src/plugins/graph/utilities/graph-utils.ts

```diff
@@ -51,7 +51,8 @@ export function ptInRect(pt: Point, iRect: Rect) {
 /**
  * This function closely follows V2's CellLinearAxisModel:_computeBoundsAndTickGap
  */
-export function computeNiceNumericBounds(min: number, max: number): { min: number, max: number } {
+export function computeNiceNumericBounds(
+  min: number|undefined, max: number|undefined): { min: number, max: number } {
```

Comment:
Note: Undefined was already getting passed in (when graph had no data), but was getting translated to NaN and throwing warnings.